Fix schemas

```
/home/travis/gopath/src/github.com/skygeario/skygear-server/pkg/auth/handler/oauth/challenge.go:58:1: invalid json schema: invalid character '"' after object key:value pair
```

We should build the API doc as part of CI pipeline for PR to avoid this in future.

--- a/.travis.yml
+++ b/.travis.yml
@@ -41,6 +41,8 @@ script:
   - git status | grep "go.mod$"; test $? -eq 1
   - make lint
   - make test
+  - make generate-doc-auth > /dev/null
+  - make generate-doc-asset > /dev/null
 
 before_deploy:
   - make generate-static-asset

--- a/Makefile
+++ b/Makefile
@@ -51,6 +51,7 @@ vendor:
 	go install github.com/golang/mock/mockgen
 	go install tools/nextimportslint.go
 	go install github.com/google/wire/cmd/wire
+	go install github.com/skygeario/openapi3-gen/cmd/openapi3-gen
 
 .PHONY: go-generate
 go-generate:

--- a/devtools/devtools.go
+++ b/devtools/devtools.go
@@ -8,7 +8,9 @@ package devtools
 // this package itself cannot compile.
 import (
 	_ "github.com/golang/mock/mockgen"
+	_ "github.com/google/wire/cmd/wire"
 	_ "github.com/mitchellh/gox"
+	_ "github.com/skygeario/openapi3-gen/cmd/openapi3-gen"
 	_ "github.com/tinylib/msgp"
 	_ "golang.org/x/tools/cmd/cover"
 	_ "golang.org/x/tools/cmd/stringer"

--- a/go.mod
+++ b/go.mod
@@ -41,6 +41,7 @@ require (
 	github.com/sfreiberg/gotwilio v0.0.0-20181012193634-a13e5b0d458a
 	github.com/sirupsen/logrus v1.4.2
 	github.com/skygeario/go-confusable-homoglyphs v0.0.0-20191212061114-e2b2a60df110
+	github.com/skygeario/openapi3-gen v0.0.0-20190808034633-90418c3d9171
 	github.com/smartystreets/assertions v1.0.0 // indirect
 	github.com/smartystreets/goconvey v0.0.0-20190731233626-505e41936337
 	github.com/tinylib/msgp v1.1.0

--- a/pkg/auth/handler/oauth/challenge.go
+++ b/pkg/auth/handler/oauth/challenge.go
@@ -60,7 +60,7 @@ const ChallengeResponseSchema = `
 	"$id": "#OAuthChallengeResponse",
 	"type": "object",
 	"properties": {
-		"token": { "type": "string" }
+		"token": { "type": "string" },
 		"expire_at": { "type": "string" }
 	},
 	"required": ["token", "expire_at"]

